index.html

```diff
@@ -2,15 +2,13 @@
 <html>
 <head>
   <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
-  <script src="bower_components/webcomponentsjs/webcomponents.js"></script>
-  <link rel="import" href="bower_components/polymer/polymer.html">
-  <link rel="import" href="bower_components/core-component-page/core-component-page.html">
-</head>
+  <script src="../webcomponentsjs/webcomponents-lite.js"></script>
+  <link rel="import" href="../iron-component-page/iron-component-page.html">
```

Comment:
Well the new recommended approach (for 1.0) to create reusable elements is to use a tool called `polyserve` which can deal with relativ links. See this short video: https://www.youtube.com/watch?v=p7Q1mQtFGM8
You can also check out the official seed-element (https://github.com/PolymerElements/seed-element/blob/master/index.html)